SmartDeviceLink/SDLTextAndGraphicManager.m

```diff
@@ -446,10 +465,72 @@ - (void)sdl_updateCurrentScreenDataFromShow:(SDLShow *)show {
 
 #pragma mark - Helpers
 
+/**
+ *  Checks if an artwork needs to be uploaded to Core.
+ *
+ *  @param artwork     The artwork to be uploaded to Core
+ *  @return            True if the artwork does not need to be uploaded to Core; false if artwork stills needs to be sent to Core.
+ */
 - (BOOL)sdl_uploadedArtworkOrDoesntExist:(SDLArtwork *)artwork {
     return (!artwork || [self.fileManager hasUploadedFile:artwork]);
 }
 
+/**
+ *  Checks if one or both artworks in a Show request need to be uploaded to Core.
+ *
+ *  @param primaryGraphic      The main graphic in the Show request
+ *  @param secondaryGraphic    The secondary graphic in the Show request
+ *  @param maxRetryCount       The max number of times the artwork is allowed to be uploaded to Core
+ *  @return                    True if all the artworks in the Show request have been either uploaded to Core or failed to upload to Core after several attempts; false if one or both of artworks stills needs to be sent to Core.
```

Comment:
This comment has grammatical errors. Also, the name of this method is misleading. It's asking "artwork upload failed?", but "YES" is "succeeded".

Reply:
Fixed. Renamed method